pkg/disk/bdf.go

```diff
@@ -359,3 +360,37 @@ func forceDetachAllowed(disk *ecs.Disk, nodeID string) (allowed bool, err error)
 	// case 2
 	return inst.Status == InstanceStatusStopped, nil
 }
+
+var vfOnce = new(sync.Once)
+
+func IsVFNode() bool {
```

Comment:
run this code when set GlobalConfigVar.DiskBdfEnable value is better. not need running for every check process.

Reply:
I think  IsVFNode is more reliable.  The configuration(GlobalConfigVar.DiskBdfEnable) is easy to cause incorrect configuration due to human problems